site/documentation/content/user-guide/coap-adapter.md

```diff
@@ -0,0 +1,231 @@
++++
+title = "CoAP Adapter"
+weight = 210
```

Comment:
IMHO we should set this to 225 so that it will appear right after the MQTT adapter